test/tests/pattern-selection.json

```
@@ -0,0 +1,150 @@
+{
+  "$schema": "https://raw.githubusercontent.com/unicode-org/message-format-wg/main/test/schemas/v0/tests.schema.json",
+  "scenario": "Pattern selection",
+  "description": "Tests for the pattern selection parts of the specification",
+  "defaultTestProperties": {
+    "locale": "en-US"
```

Does this need to be `en-US`? Should this be `und`? `zxx`? We shouldn't over-favor US English, especially when the tests themselves are not locale-affected.

---

This can be anything.

```suggestion
    "locale": "und"
```